### DISCUSSION TITLE
Syntax error with new update

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#850] [Fixed Issues]
![Screenshot 2022-03-17 080443](https://user-images.githubusercontent.com/93144100/158755060-885f4c06-9aab-4099-bb31-46de7404212e.jpg)

**Comment by ceejayemm** [comment]
Same as #849 ?

**Comment by hjfisch** [comment]
Solved ,working now.
Sorry for Doublepost.
